screens/matches/match-detail.tsx

```diff
@@ -1,12 +1,81 @@
-import { ProcessedMatch } from "../../src/coh3/coh3-types";
+import { FactionSide, ProcessedMatch } from "../../src/coh3/coh3-types";
+import { Container, Flex, Title, Text, Box } from "@mantine/core";
+import { matchTypesAsObject } from "../../src/coh3/coh3-data";
+import RenderMap from "../players/tabs/recent-matches-tab/matches-table/render-map";
+import { getMatchPlayersByFaction } from "../../src/coh3/helpers";
+import PlayerMatchesDataTable from "../../components/PlayerMatchesDataTable";
+
+export default function Component({ matchData }: { matchData: ProcessedMatch }) {
+  const matchtype_id = matchData.matchtype_id;
+  const matchType =
+    matchTypesAsObject[matchtype_id as number]["localizedName"] ||
+    matchTypesAsObject[matchtype_id as number]["name"] ||
+    "unknown";
+
+  const formatDuration = (durationInSeconds: number) => {
+    const hours = Math.floor(durationInSeconds / 3600);
+    const minutes = Math.floor((durationInSeconds % 3600) / 60);
+    const seconds = durationInSeconds % 60;
+    return `${String(hours).padStart(2, "0")}:${String(minutes).padStart(2, "0")}:${String(seconds).padStart(2, "0")}`;
+  };
+
+  const getMatchResult = (matchData: ProcessedMatch) => {
+    let winner: FactionSide;
+    let loser: FactionSide;
+    winner = "axis";
```

Comment:
It might be better to have here winner = null and loser = null, so you know you are not mistakenly targeting the results.